src/form/sections/SchedulingMenu.tsx

```diff
@@ -0,0 +1,145 @@
+import {
+  Col,
+  MultiSelection,
+  MultiSelectionDefaultOptionType,
+  Row,
+  Select,
+  TextField,
+  Timepicker,
+} from '@folio/stripes/components';
+import React from 'react';
+import { Field, useField } from 'react-final-form';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { Weekday, useLocaleWeekdays } from '../../utils/weekdayUtils';
+import SchedulingFrequency from '../../types/SchedulingFrequency';
+
+export function getIntervalLabel(frequency: SchedulingFrequency) {
```

Comment:
instead of this function you could just use 

`<FormattedMessage id=`ui-plugin-bursar-export.bursarExports.scheduling.interval.${frequency}` />`

where translations in en.json could be `bursarExports.scheduling.interval.HOURS` etc